app/frontend/src/components/form/Location.vue

```diff
@@ -0,0 +1,392 @@
+        </v-col>
+      </v-row>
+    </div>
+
+    <v-checkbox v-model="accMotel" :readonly="reviewMode" label="Motel / Hotel in town"></v-checkbox>
+    <div v-if="accMotel">
+      <v-row>
+        <v-col cols="12" sm="6" lg="5">
+          <label>Name</label>
+          <v-text-field v-model="motelName" dense flat outlined solo />
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-col cols="12" sm="6" lg="5">
+          <label>Address line 1</label>
+          <v-text-field v-model="motelAddressLine1" dense flat outlined solo />
+        </v-col>
+
+        <v-col cols="12" sm="6" lg="5">
+          <label>Address line 2 (Optional)</label>
+          <v-text-field v-model="motelAddressLine2" dense flat outlined solo />
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-col cols="12" sm="6" lg="5">
+          <label>City</label>
+          <v-text-field v-model="motelCity" dense flat outlined solo />
+        </v-col>
+        <v-col cols="12" sm="3" lg="2">
+          <label>Province</label>
+          <v-select
+            dense
+            flat
+            outlined
+            solo
+            single-line
+            label="select"
+            v-model="motelProvince"
+            :items="provinces"
+          />
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-col cols="12" sm="3" lg="2">
+          <label>Postal Code</label>
+          <v-text-field v-model="motelPostalCode" dense flat outlined solo />
+        </v-col>
+      </v-row>
+    </div>
+
+    <v-checkbox v-model="accWorkersHome" :readonly="reviewMode" label="Worker's home in community"></v-checkbox>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapMutations } from 'vuex';
+
+import CityLookup from '@/components/form/CityLookup.vue';
+
+export default {
+  name: 'Location',
+  props: {
+    reviewMode: Boolean
+  },
+  components: {
+    CityLookup,
+  },
+  data() {
+    return {
+      startDateMenu: false,
+      endDateMenu: false,
+
+      // Todo: constants file
+      provinces: [
+        'AB',
+        'BC',
+        'MB',
+        'NB',
+        'NL',
+        'NS',
+        'NT',
+        'NU',
+        'ON',
+        'PE',
+        'QC',
+        'SK',
+        'YT'
+      ],
+
+      // Location
+      licenceesRules: [v => !!v || 'Name of licencee(s) required'],
+      startDateRules: [v => !!v || 'Start date is required'],
+      endDateRules: [v => !!v || 'End date is required'],
+      locationCityRules: [
+        v => !!v || 'Closest Community / Town / City is required'
+      ],
+      // Todo, put in some utility fxn somewhere if needed again
+      numberOfWorkersRules: [
+        v => new RegExp('^[-+]?\\d+$').test(v) || 'invalid # of workers',
+        v => v > 0 || '# of workers must be greater than 0',
+        v => v < 9999 || '# of workers must 9999 or less'
+      ]
+    };
+  },
+  computed: {
+    ...mapGetters('form', ['location']),
```

Comment:
The one concern I have now that I think about this, is if we're using this Location component in multiple places, but it's pulling from not necessarily the right store. This could introduce a subtle and hard to trace bug - would need to double check that behaviors on all pathways are doing what you think it should be doing.